.gitignore

```diff
@@ -9,3 +9,6 @@ data/
 *.bloop
 *.metals
 .bsp
+
+# dependencies
+/node_modules
```

Comment:
TODO: po usunięciu `package.json` z głównego katalogu to będzie też do usunięcia